src/Bindings/LuaJson.cpp

```diff
@@ -134,7 +134,7 @@ static Json::Value JsonSerializeTable(cLuaState & a_LuaState)
 	{
 		if (lua_type(a_LuaState, -2) == LUA_TNUMBER)
 		{
-			int idx;
+			int idx = 0;
 			a_LuaState.GetStackValue(-2, idx);
```

Comment:
I don't think it's needed, because we've already checked that it's a number, so it's always read successfully (unless someone else bombs our memory).
I guess I could rewrite this to instead depend on the `GetStackValue()`'s return as to whether the value is an int or a string